src/main/cpp/PlayerLine.cpp

```diff
@@ -7,41 +7,54 @@ PlayerLine::PlayerLine()
 	
 }
 
-int PlayerLine::getNumPlayers() const 
+// PlayerLine::~PlayerLine() {
```

Comment:
Does the shared_ptr automatically deallocate or do we need to add a deconstructor that deallocates it.

Reply:
Yes it will automatically de-allocate once there are no longer any references to it.